src/app/components/profile/modal/skills/skills.scss

```diff
@@ -2,72 +2,75 @@
 @import "styles/_colors";
 
 .skills-modal {
-    .edit-skills-list {
-        min-height: 420px;
-        max-height: 420px;
-        overflow: hidden;
-        max-width: 100%;
-        zoom: 1;
-        .edit-skills-list-item {
-            width: 100%;
-            zoom: 1;
+        .edit-skills-list {
+            min-height: 420px;
+            max-height: 420px;
             overflow: hidden;
-            text-overflow: clip;
-            white-space:nowrap;
-            .item-content {
-                display: flex;
-                flex-flow: row wrap;
-            }
-            .item-buttons {
-                width: 40px;
-                min-width: 40px;
-            }
-            .deletebutton {
-                @include transition(all 0.3s ease);
-                min-width: 100%;
-                max-width: 100%;
-                display: inline-block;
-                color: rgba(255, 255, 255, 1);
-                background-color: red;
-                text-align: center;
-                line-height: 58px;
-                font-size: 20px;
-                cursor: pointer;
-            }
-            .editbutton {
-                @include transition(all 0.3s ease);
-                min-width: 100%;
-                max-width: 100%;
-                display: inline-block;
-                color: rgba(255, 255, 255, 1);
-                background-color: #337ab7;
-                text-align: center;
-                line-height: 58px;
-                font-size: 20px;
-                cursor: pointer;
-            }
-            .name {
-                font-weight: bold;
-                flex-basis: 100%;
-            }
-            .level {
-                width: 70px;
-                flex-basis: 33%;
-            }
-            .years {
-                flex-basis: 33%;
-            }
-            .futureLevel {
-                width: 100px;
-                flex-basis: 33%;
-            }
+            max-width: 100%;
+            zoom: 1;
+            .edit-skills-list-item {
```

Comment:
where does .edit-skills-list-item { end? I belive you have the wrong indentation or have close the block wrong.